bundles/test/org.eclipse.smarthome.magic/ESH-INF/thing/thing-types.xml

```diff
@@ -101,4 +102,29 @@
 			<channel id="number" typeId="number" />
 		</channels>
 	</thing-type>
+
+	<thing-type id="firmware-update">
+		<label>Magic Firmware Updatable Thing</label>
+		<description>A thing that can be firmware updated.</description>
+
+		<properties>
+			<property name="firmwareVersion">0.1.0</property>
+		</properties>
+
+		<config-description uri="thing-type:magic:update-config">
+			<parameter name="updateModel" type="text" required="true">
+				<required>true</required>
+				<options>
+					<option value="Not specified">Not specified</option>
```

Comment:
Wouldn't it make sense to have something different as values, like e.g. numbers? Just a thought, to keep it a little more realistic...

Reply:
Yes, will change them a bit

Reply:
Little misunderstanding, sorry... I was rather referring to something like this:

```xml
<option value="1">Abracadabra</option>
<option value="2">Alohomora</option>
<option value="3">Lumos</option>
```

Reply:
OK, I will ca choose some Harry Potter spells, starting with different letter to make them distinguishable :relaxed: And will remove the "not specified"  value, we will just have one model without specific firmwares.